Realm/Realm.Sync/Native/SyncConfiguration.cs

```diff
@@ -50,7 +50,7 @@ internal string Url
         }
 
         [MarshalAs(UnmanagedType.I1)]
-        internal bool client_validate_ssl;
+        internal bool EnableSSLValidation;
```

Comment:
Why the rename?

Reply:
Just bugged me a little that half the public members are pascal case, the others - snake case. Will revert though, as you're right - better to stick to the c++ naming for matching fields.